### ISSUE TITLE
[v2] Autocomplete: Uncaught TypeError: Cannot read property '0' of undefined in framework7.js:27256 

### ISSUE THREAD

**Issue by byronics** [framework7io/framework7#2171]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Framework7 Version: 2.0.6
* Platform and Target: iOS & Android (issue only happens in Android/MD layout)

### What you did
Created an Autocomplete dropdown

**Layout**
```
<form class="searchbar" id="people-searchbar">
    <div class="searchbar-inner">
        <div class="searchbar-input-wrap">
            <input type="search" placeholder="Search people" autocorrect="off" autocapitalize="off" onkeypress="return event.keyCode != 13;"/>
            <i class="searchbar-icon"></i>
            <span class="input-clear-button"></span>
        </div>
        <span class="searchbar-disable-button dark">Cancel</span>
    </div>
</form>
```

**Init**
```
myPeopleSearch = app.autocomplete.create({
    inputEl: '#people-searchbar input[type="search"]',
    openIn: 'dropdown',
    preloader: true, 
    valueProperty: 'u_id', 
    textProperty: 'display', 
    limit: 20,
    dropdownPlaceholderText: 'Search by Username or Full Name',
    source: function (query, render) {
   ...
```

### Expected Behavior
No errors in JS console

### Actual Behavior
Component is working, but when loaded in Android/MD, every click in the app shows an error in the JS console, 1 error per Autocomplete instance on every click:

```
framework7.js:27256 Uncaught TypeError: Cannot read property '0' of undefined
    at HTMLHtmlElement.s (framework7.js:27256)
    at HTMLHtmlElement.a (framework7.js:966)
    at framework7.js:3936
    at i.<anonymous> (framework7.js:4126)
    at framework7.js:2909
    at Array.forEach (<anonymous>)
    at framework7.js:2908
    at Array.forEach (<anonymous>)
    at i.se.emit (framework7.js:2902)
    at p (framework7.js:4203)
```

This error does not happen in iOS layout.

Basically, framework7.js:27256 is `if ($targetEl.is(ac.$inputEl[0]) || ($targetEl.closest(ac.$dropdownEl[0]).length)) { return; }`, so it's looking for [0] on an element that doesn't exist...not sure why this issue doesn't manifest in iOS layout, perhaps the dynamic dropdownEl is handled differently,

**Comment by bmcbride** [comment]
I'm noticing this behavior as well.

**Comment by owen-mills** [comment]
Same issue

**Comment by byronics** [comment]
@owen-mills I updated to v2.0.8 and the issue went away.
https://github.com/framework7io/framework7/releases/tag/v2.0.8
